### ISSUE TITLE
Subtitle (from YAML header) not appearing in PDF when converting from markdow

### ISSUE THREAD

**Issue by dylan-chong** [jgm/pandoc#3896]
Pandoc: 1.19.2.1, Mac OS X 10.11.6

##  markdown file (f.md)

```
---
title: Title
subtitle: Subtitle
author:
    - I am an author
date: \today{}
---

I am text
```

## Output command

`pandoc f.md -o f.pdf`

## Result

The subtitle is not shown in the PDF

<img width="637" alt="screen shot 2017-09-06 at 7 10 27 pm" src="https://user-images.githubusercontent.com/17482349/30098683-118b9b78-9337-11e7-8040-f2e98e015fd4.png">

**Comment by mb21** [comment]
From the [MANUAL](http://pandoc.org/MANUAL.html#variables-set-by-pandoc):

> document subtitle, included in HTML, EPUB, LaTeX, ConTeXt, and Word docx; renders in LaTeX only when using a document class that supports \subtitle, such as beamer or the KOMA-Script series (scrartcl, scrreprt, scrbook)

**Comment by stevegt** [comment]
Thank You!  Saved my life -- several hours of it, anyway.

You see, I'm one of them thar UNIX sysadmin types with the requisite 30 years of crotchety experience hacking around in arcane voodoo.  I, too, saw that same paragraph in the manual yesterday, and I've spent the intervening hours trying to figure out what the heck a user is supposed to do.  Latex docs, stack exchange, and the intertubes at large aren't much help with something this simple -- most latex conversation is between people writing masters theses.  

This is the only example I've been able to find that shows how to do this.  It might be worth adding it to that paragraph in the manual.
